libs/flags/yaml_flag.go

```diff
@@ -0,0 +1,37 @@
+package flags
+
+import (
+	"fmt"
+	"os"
+	"sigs.k8s.io/yaml"
+)
+
+type YamlFlag struct {
+	raw []byte
+}
+
+func (y *YamlFlag) String() string {
+	return fmt.Sprintf("YAML (%d bytes)", len(y.raw))
+}
+
+// TODO: Command.MarkFlagFilename()
+func (y *YamlFlag) Set(v string) error {
+	// Load request from file
+	buf, err := os.ReadFile(v)
```

Comment:
This means the flag is actually a file flag. The JSON flag accepts both raw JSON or a JSON file name, but the latter requires a prefix of `@` to indicate it is a file name (similar to how curl does this).

Up to you how you want to deal with this. I prefer symmetry with the behavior of the JSON flag, or otherwise you can rename this to `YamlFileFlag`, store the filename, and read the file on demand.

Reply:
so in our case, the flags are `--manifest` and `--resource` would `--@manifest` be preferred if we are dealing exclusively with files?

Reply:
I mean an `@` prefix on the value that's set.

Also see: https://github.com/databricks/cli/blob/5a8cd0c5bc3f12805019b16bcede8d6edd7d5697/libs/flags/json_flag.go#L18-L30